src/brogue/IO.c

```diff
@@ -1333,6 +1333,11 @@ void getCellAppearance(short x, short y, enum displayGlyph *returnChar, color *r
         cellChar = G_WALL_TOP;
     }
 
+    // Show granite rather than brick on the opposite side of walls
+    if (cellChar == G_WALL && (!coordinatesAreInMap(x, y+1) || pmap[x][y+1].layers[DUNGEON] == GRANITE)) {
```

Comment:
Using `pmap` here leaks information about what is below wall tiles. And if we do it based on player knowledge I think it will be jarring as tiles switch from granite to walls as the player sees more of the map. So, it's a nice idea, but I think it's not suited.